### PR TITLE
refactor: set gas limit to 2.8M for migrateAll and 2M for all crvAAVE vaults

### PR DESCRIPTION
**Pull request by x48-crypto**
0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab
0xb4D1Be44BfF40ad6e506edf43156577a3f8672eC
0xBacB69571323575C6a5A3b4F9EEde1DC7D31FBc1

Withdraw All/Withdraw: 2M gas
Migrate: 2.8M gas

### CODE REVIEW

File: app/containers/Vaults/saga.js [hunk 1]
```diff
@@ -134,7 +135,9 @@ function* withdrawFromVault(action) {
 
   const vaultAddress = _.get(vaultContractData, 'address');
   const vaultIsAave =
```
Comment (x48-crypto) [inline app/containers/Vaults/saga.js:137]: Withdraw All

File: app/containers/Vaults/saga.js [hunk 2]
```diff
@@ -223,7 +226,9 @@ function* withdrawAllFromVault(action) {
...
-    vaultAddress === crvAaveAddress || vaultAddress === crvSAaveAddress;
+    vaultAddress === crvAaveAddress ||
+    vaultAddress === crvSAaveV2Address ||
+    vaultAddress === crvSAaveV1Address;
```
Comment (x48-crypto) [inline app/containers/Vaults/saga.js:232]: Withdraw

File: app/containers/Vaults/saga.js [hunk 3]
```diff
@@ -445,6 +456,20 @@ function* migrateVault(action) {
...
+        vaultMigrationData.vaultFrom,
+        vaultMigrationData.vaultTo,
+        {
+          from: account,
+          gas: 2800000,
+        },
+      );
+      return;
+    }
+
```
Comment (x48-crypto) [inline app/containers/Vaults/saga.js:472]: Migrate

**Review by xgambitox — APPROVED** [review]

**Comment by dudesahn** [comment]
lgtm

**Review by dudesahn — APPROVED** [review]

**Pull request merged**